### PR TITLE
AL-169

### PR DESCRIPTION
**Pull request by c4rt4nya**

### CODE REVIEW

File: src/main/java/com/directv/lcms/service/MultiViewerService.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package com.directv.lcms.service;
 
 import com.directv.lcms.dto.*;
+import com.directv.lcms.rest.controller.EncoderRequest;
```
Comment (damianChapman) [inline src/main/java/com/directv/lcms/service/MultiViewerService.java:9]: Probably better not to import everything here. There is an intellj IDE setting(see Code Style/Java) to stop this from happening.

File: src/main/java/com/directv/lcms/service/MultiViewerService.java [hunk 2]
```diff
@@ -66,6 +67,9 @@
     @Value("${tag.audio.pids.statistics.url}")
     private String audioPidsStatisticsUrl;
 
+    @Value("${tag.encoder.id}")
```
Comment (damianChapman) [inline src/main/java/com/directv/lcms/service/MultiViewerService.java]: Is this needed now as we have multiple encoders configured?

File: src/main/java/com/directv/lcms/rest/controller/MultiViewerControllerV1.java [hunk 3]
```diff
@@ -178,6 +178,14 @@ private ResponseEntity deleteMultiViewer(@PathVariable String name) {
         }
     }
 
+    @RequestMapping(value = "/multiviewer/tag/encoder/layout/{id}", method = RequestMethod.PUT, produces = "application/json; charset=UTF-8")
+    @ApiOperation(value = "Updates the encoder to associate it with a layout")
+    private ResponseEntity updateMultiViewer(@ApiParam(value = "Layout id")
+                                             @PathVariable String id) {
+        return multiViewerService.updateLayoutofEncoder(id);
```
Comment (damianChapman) [inline src/main/java/com/directv/lcms/rest/controller/MultiViewerControllerV1.java]: updateEncodeLayout may be a better method name?

**Review by damianChapman — CHANGES_REQUESTED** [review]

File: src/main/java/com/directv/lcms/service/MultiViewerService.java (hunk 2 above)
Comment (c4rt4nya) [reply src/main/java/com/directv/lcms/service/MultiViewerService.java]: done

File: src/main/java/com/directv/lcms/service/MultiViewerService.java (hunk 1 above)
Comment (c4rt4nya) [reply src/main/java/com/directv/lcms/service/MultiViewerService.java:9]: done

Comment (c4rt4nya) [reply src/main/java/com/directv/lcms/service/MultiViewerService.java]: done

**Review by damianChapman — APPROVED** [review]

**Pull request merged**